core/modules/qdisp/QueryResource.h

```diff
@@ -58,7 +58,7 @@ class QueryResource : public XrdSsiService::Resource {
                   ExecStatus& status)
         : Resource(rPath.c_str()),
           _session(NULL),
-          _request(NULL),
+          _request(),
```

Comment:
Why this change?  _request should be explicitly init'd since it is a raw pointer